src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post" 
+    >
+      <fieldset class="fieldset" name="1">
+        <legend>Personal information</legend>
+        <label class="form-field">
+          Surname: 
+          <input 
+            type="text" 
+            autocomplete="off" 
+            minlength="3" 
+            maxlength="32"
+            name="inf"
+          >
+        </label>
+        <label class="form-field">
```

Comment:
You need to add spaces where needed. Check all file please
```suggestion
        </label>
        
        <label class="form-field">
```